src/services/ui/src/features/package-actions/index.tsx

```diff
@@ -52,6 +57,14 @@ export const packageActionRoutes: RouteObject = {
       element: <RespondToRai />,
       action: respondToRaiSubmission,
     },
+    {
+      path: "1915b-capitated-amendment",
+      element: <Capitated1915BWaiverAmendmentPage />,
+    },
+    {
+      path: "1915b-contracting-amendment",
+      element: <Contracting1915BWaiverAmendmentPage />,
+    },
```

Comment:
Wiring the same form pages into the Actions router to make use of the "/action/:authority/:id" path